postgres/datadog_checks/postgres/explain_parameterized_queries.py

```diff
@@ -157,12 +157,13 @@ def _deallocate_prepared_statement(self, dbname, query_signature):
             )
 
     def _execute_query(self, dbname, query):
-        with self._check._get_db(dbname).cursor(cursor_factory=psycopg2.extras.DictCursor) as cursor:
-            logger.debug('Executing query=[%s]', query)
-            cursor.execute(query)
+        with self._check.db_pool.get_connection(dbname, self._check._config.idle_connection_timeout) as conn:
```

Comment:
does the connection need to be wrapped with a `with` here? I thought the plan was to only close this if we had to (aka when the pool is full & we want a new connection)...

should this code instead be:

```
with self._check.db_pool.get_connection(dbname, self._check._config.idle_connection_timeout).cursor() as cursor:
```